extensions/amp-consent/amp-consent.md

```diff
@@ -114,10 +117,9 @@ AMP expects the response to be a JSON object like the following:
 
 ```html
 { 
-  "promptIfUnknown": {boolean}, // deprecated
-  "collectConsent": {boolean} [default: false], // (new) whether to prompt consent dialog and collect consent
-  "consentState": {enum} [default: null], (new) // (new) the latest consent state known by the server
-  "updateCache": {boolean} [default: true] // (new) whether to update client cache with the new value
+  "promptIfUnknown": {boolean}, // instructs AMP whether to collect consent
+  "consentState": {enum} [default: null], // (new) the latest consent state known by the server
+  "useCache": {boolean} [default: true] // (new) whether to store consent state to client cache to avoid blocking request for user's next visit. Set to `false` if strict server-client syncing is required.
```

Comment:
`useCache` and `updateCache` is a bit different to me. `useCache: false` to me sounds like erase any stored value if exists. If we are only updating cache, I do think `updateCache` is a better name